src/utils/path2d.c

```diff
@@ -175,8 +175,8 @@ GF_Err gf_path_close(GF_Path *gp)
 	end = gp->points[gp->n_points-1];
 	end.x -= start.x;
 	end.y -= start.y;
-	diff = gf_mulfix(end.x, end.x) + gf_mulfix(end.y, end.y);
-	if (ABS(diff) > FIX_ONE/1000) {
+	//diff = gf_mulfix(end.x, end.x) + gf_mulfix(end.y, end.y);
+	if (FIX_ONE < ABS(end.x) || FIX_ONE < ABS(end.y)) { // ABS(diff) > FIX_ONE/1000
```

Comment:
I think using FIX_ONE here is wrong, we could end up not closing a path while the old code was. Could you replace with FIX_ONE/100 ?

Reply:
done